community/detectors/slurm_exposed_rest_api/src/test/java/com/google/tsunami/plugins/detectors/rce/SlurmExposedRestApiDaemonVuLnDetectorTest.java

```diff
@@ -0,0 +1,185 @@
+          Arrays.fill(bytes, (byte) 0xFF);
+        }
+      };
+
+  @Before
+  public void setUp() throws IOException {
+    mockCallbackServer.start();
+    Guice.createInjector(
+            new FakeUtcClockModule(fakeUtcClock),
+            new HttpClientModule.Builder().build(),
+            FakePayloadGeneratorModule.builder()
+                .setCallbackServer(mockCallbackServer)
+                .setSecureRng(testSecureRandom)
+                .build(),
+            new SlurmExposedRestApiDaemonDetectorBootstrapModule())
+        .injectMembers(this);
+  }
+
+  @After
+  public void tearDown() throws Exception {
+    mockTargetService.shutdown();
+    mockCallbackServer.shutdown();
+  }
+
+  @Test
+  public void detect_whenVulnerable_returnsVulnerability() throws IOException {
+    startMockWebServer(true);
+    mockCallbackServer.enqueue(PayloadTestHelper.generateMockSuccessfulCallbackResponse());
+
+    DetectionReportList detectionReports =
+        detector.detect(targetInfo, ImmutableList.of(targetNetworkService));
+
+    assertThat(detectionReports.getDetectionReportsList())
+        .containsExactly(
+            DetectionReport.newBuilder()
+                .setTargetInfo(targetInfo)
+                .setNetworkService(targetNetworkService)
+                .setDetectionTimestamp(
+                    Timestamps.fromMillis(Instant.now(fakeUtcClock).toEpochMilli()))
+                .setDetectionStatus(DetectionStatus.VULNERABILITY_VERIFIED)
+                .setVulnerability(
+                    Vulnerability.newBuilder()
+                        .setMainId(
+                            VulnerabilityId.newBuilder()
+                                .setPublisher("TSUNAMI_COMMUNITY")
+                                .setValue("SlurmRestApiDaemonRce"))
+                        .setSeverity(Severity.CRITICAL)
+                        .setTitle("Exposed Slurm REST API Server")
+                        .setDescription(
+                            "This detector checks for exposed slurm rest api servers by submitting a job "
+                                + "and checking a callback response on tsunami callback server")
+                        .setRecommendation(
+                            "set proper authentication method for slurm rest api server or "
+                                + "remove the public access to rest api"))
+                .build());
+    Truth.assertThat(mockTargetService.getRequestCount()).isEqualTo(2);
+    Truth.assertThat(mockCallbackServer.getRequestCount()).isEqualTo(1);
+  }
+
+  @Test
+  public void detect_ifNotVulnerable_doesNotReportVuln() throws IOException {
+    startMockWebServer(false);
+    DetectionReportList detectionReports =
+        detector.detect(targetInfo, ImmutableList.of(targetNetworkService));
+    assertThat(detectionReports.getDetectionReportsList()).isEmpty();
+    Truth.assertThat(mockTargetService.getRequestCount()).isEqualTo(2);
```

Comment:
```suggestion
   assertThat(mockTargetService.getRequestCount()).isEqualTo(2);
```
Be consistent with the others usages of `assertThat`